package.json

```diff
@@ -47,6 +50,8 @@
     "redux-form": "^7.0.1",
     "redux-immutable-state-invariant": "^2.0.0",
     "redux-logger": "^3.0.6",
+    "redux-mock-store": "^1.2.3",
+    "redux-persist": "^4.8.3",
```

Comment:
Yeah I did pull it off, it must have been crossed in on accident. I can remove it.